asyncapi-cli/src/main/java/io/ballerina/asyncapi/cmd/websockets/AsyncAPIDiagnostic.java

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.asyncapi.cmd.websockets;
+
+import io.ballerina.tools.diagnostics.Diagnostic;
+import io.ballerina.tools.diagnostics.DiagnosticInfo;
+import io.ballerina.tools.diagnostics.DiagnosticProperty;
+import io.ballerina.tools.diagnostics.Location;
+import io.ballerina.tools.text.LinePosition;
+import io.ballerina.tools.text.LineRange;
+
+import java.text.MessageFormat;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Represents a {@code Diagnostic} related to asyncapi command.
+ *
+ */
+public class AsyncAPIDiagnostic extends Diagnostic {
+    private final DiagnosticInfo diagnosticInfo;
+    private final Location location;
+    private final List<DiagnosticProperty<?>> properties;
```

Comment:
Could you please explain the reason for using wild-card here ?

Reply:
Seems like this property is not accessed anywhere in the code. Therefore I'll remove this.